network/p2p/test/sporking_test.go

```diff
@@ -202,10 +201,11 @@ func TestOneToKCrosstalkPrevention(t *testing.T) {
 		"test_one_to_k_crosstalk_prevention",
 		idProvider,
 	)
-	idProvider.On("ByPeerID", node1.Host().ID()).Return(&id1, true).Maybe()
+
+	// idProvider.On("ByPeerID", node1.Host().ID()).Return(&id1, true).Maybe()
```

Comment:
you can remove this now

Reply:
https://github.com/onflow/flow-go/pull/4524/commits/2962ffca4e1be1a5b5f54ca2ac8c8c93c487a509